js/add-field.js

```diff
@@ -44,12 +44,16 @@
           !existing_values.includes(element.asurite_id.raw + ':' + deptid)) {
 
           var department_data = getDepartmentData(element, deptid);
+          let title = department_data['title'];
+          if (title == null) {
```

Comment:
If you is expected to be exactly null, this should be 3 equals otherwise Javascript will type coerce anything falsy and it will equal it. This can be `"", 0, undefined, or NaN`. Otherwise if you just want to check any of those values as title, this will work

Reply:
Cool. I think I'm okay with a falsy in this case, because the data might include an empty string.